tests/installation/opensuse_welcome.pm

```diff
@@ -15,7 +15,20 @@ use utils;
 use x11utils 'handle_welcome_screen';
 
 sub run {
-    handle_welcome_screen;
+    my $upgrade = get_var('UPGRADE');
+    if ($upgrade == 1) {
+        send_key 'esc';
```

Comment:
What does this escape do?

Reply:
I'm working on that, in order to avoid this situation : https://openqa.opensuse.org/tests/2264087#step/opensuse_welcome/4

Reply:
That's for GNOME activities handling, the `opensuse-welcome-gnome40-activities` tag is used to detect that and there's also `handle_gnome_activities`.